src/Controller/StatusController.php

```diff
@@ -61,8 +63,23 @@ public function statusAction(Request $request)
     private function verifyNoDatabaseCacheIssues(): bool
     {
         try {
-            $pid = new Pid('b006m86d'); //Eastenders
-            $this->get('pps.programmes_service')->findByPidFull($pid);
+            // Eastenders clip
+            $clipPid = new Pid('p04r0jcv');
+            $this->get('pps.programmes_service')->findByPidFull($clipPid);
+
+            // Broadcast
+            $fromDateTime = new DateTimeImmutable('2010-01-15 06:00:00');
+            $toDatetime = new DateTimeImmutable('2017-10-16 06:00:00');
+            $sid = new Sid('bbc_radio_two');
+            $this->get('pps.broadcasts_service')->findByServiceAndDateRange($sid, $fromDateTime, $toDatetime, 1, 1);
```

Comment:
You don't need the final `1` do you?

Reply:
It will be set to 1 by default if not specified, personal perfence I think.